automation_py/webhooks/server.py

```
@@ -0,0 +1,2024 @@
+
+        # Log do job antes de salvar
+        if "website_url" in job_data:
+            logger.info(
+                f"[DEBUG] Job {job_id} antes de salvar: website_url={job_data.get('website_url')}")
+        if "result" in job_data and "site_url" in job_data.get("result", {}):
+            logger.info(
+                f"[DEBUG] Job {job_id} antes de salvar: result.site_url={job_data.get('result', {}).get('site_url')}")
+
+        # Remover o campo website_url quando o job estiver completo e tiver o campo site_url no resultado
+        if status == "completed" and result and "site_url" in result:
+            if "website_url" in job_data:
```

Ruff (0.11.9)

Use a single `if` statement instead of nested `if` statements

(SIM102)